lib/rubygems/gemcutter_utilities.rb

```diff
@@ -269,6 +271,24 @@ def get_scope_params(scope)
     scope_params
   end
 
+  def get_rubygem_scope_params(scope_params)
+    rubygem_scope_params = {}
+    return rubygem_scope_params unless gem_scope_compatible_api_scope_enabled?(scope_params)
+
+    say "Enter the name of the gem you want to scope this key to."
+    rubygem_name = ask "Gem scope [All gems]: "
```

Comment:
Is this signalling that the default behaviour/just hitting enter is to scope to all gems? I don't use many CLIs so I'm wondering if this is the standard format for declaring this behaviour, or if something like [Default: all gems] would be better